helios-client/src/test/java/com/spotify/helios/common/JobValidatorTest.java

```diff
@@ -169,6 +169,7 @@ public void testValidHostnamesPass() {
   @Test
   public void testValidVolumesPass() {
     final Job j = Job.newBuilder().setName("foo").setVersion("1").setImage("foobar").build();
+    assertThat(validator.validate(j.toBuilder().addVolume("/foo", "bar").build()), is(empty()));
```

Comment:
could you add a test of `.addVolume("/foo", "")` too? (if it is not covered elsewhere)